docs/user-guide/102-configuration.md

```diff
@@ -476,6 +476,20 @@ node05.linux-bridge-maxunavailable   Progressing
 node06.linux-bridge-maxunavailable   Pending
 ```
 
+# Component Placement
+
+In NMState, you can constrain controllers to be assigned to specific nodes using component placement. There are several options:
+
+- **NodeSelector**
+    - Optional selector. If specified, handler will run only on nodes that have matching key-value pairs as labels applied to the node.
+- **Tolerations**
+    - Optional selector. Tolerations work together with taints to ensure that handler is not scheduled onto inappropriate nodes. One or more taints are applied to a node; this marks that the node should not accept any handler that do not tolerate the taint.
+- **InfraNodeSelector, InfraTolerations**
+    - Optional selectors. They have similar purpose as NodeSelector and Tolerations, respectively. However, they restrict webhook and certmanager instead of handler.
```

Comment:
Optional ~~selectors~~

It may be better to list both fields separately, with relevant description